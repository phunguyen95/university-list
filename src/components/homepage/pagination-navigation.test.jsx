import { render, screen } from "@testing-library/react";
import PaginationNavigation from "./pagination-navigation";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";

test("renders pagination navigaton view without crashing", () => {
    const history = createMemoryHistory();

  const { asFragment } = render(<Router history={history}><PaginationNavigation /></Router>);

  expect(asFragment()).toMatchSnapshot();
});
