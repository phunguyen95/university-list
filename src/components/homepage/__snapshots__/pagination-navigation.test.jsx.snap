// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pagination navigaton view without crashing 1`] = `
<DocumentFragment>
  <nav
    aria-label="pagination"
    class="pagination"
    role="navigation"
  >
    <a
      class="pagination-previous"
      disabled=""
      title="This is the first page"
    >
      Previous
    </a>
    <a
      class="pagination-next"
    >
      Next page
    </a>
    <ul
      class="pagination-list"
    >
      <li>
        <a
          aria-current="page"
          aria-label="Page 1"
          class="pagination-link is-current"
        >
          1
        </a>
      </li>
      <li>
        <a
          aria-label="Goto page 2"
          class="pagination-link"
        >
          2
        </a>
      </li>
      <li>
        <a
          aria-label="Goto page 3"
          class="pagination-link"
        >
          3
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
